src/plugins/orangehrmDashboardPlugin/Controller/DashboardController.php

```diff
@@ -22,18 +22,39 @@
 use OrangeHRM\Core\Controller\AbstractVueController;
 use OrangeHRM\Core\Vue\Component;
 use OrangeHRM\Framework\Http\Request;
+use OrangeHRM\Core\Service\ModuleService;
 use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
 use OrangeHRM\Core\Helper\VueControllerHelper;
+use OrangeHRM\Core\Traits\ServiceContainerTrait;
+use OrangeHRM\Framework\Services;
 
 class DashboardController extends AbstractVueController
 {
     use AuthUserTrait;
+    use ServiceContainerTrait;
+
+    /**
+     * @return ModuleService
+     */
+    public function getModuleService(): ModuleService
+    {
+        return $this->getContainer()->get(Services::MODULE_SERVICE);
+    }
 
     public function preRender(Request $request): void
     {
         $component = new Component('view-dashboard');
         $this->setComponent($component);
 
+        $isLeaveModuleEnabled = false;
+
+        foreach ($this->getModuleService()->getModuleList() as $module) {
+            if ($module->getName() === 'leave') {
+                $isLeaveModuleEnabled = $module->getStatus();
+                break;
+            }
+        }
+
```

Comment:
TODO

Reply:
maybe we can include a function in ModuleDao just to get status of time and leave modules? then no need to iterate through all modules to find status. I wrote a function like that in QuickLaunchDao maybe we can move that to ModuleDao https://github.com/orangehrm/orangehrm/pull/1432/files#diff-b2a36a92a53f8851fd674c42e2b0c4c33cb501eab3a4058eb41273f3ee962aa2R90-R101